ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -100,11 +101,20 @@ export default function CreatePassword({
       </span>,
     ]);
 
+    if (passwordInput.length < 8) {
```

Comment:
Can this magic number be broken out into a configuration field or at least named constant like `PASSWORD_MIN_LENGTH`?